docs/src/index.md

```diff
@@ -39,11 +39,13 @@ log_plot!
 step_logger!
 ```
 
-These primitives can be used in implementations of [`train!`](@ref), [`evaluate!`](@ref), and [`predict!`](@ref), as well as in:
+These primitives can be used in implementations of [`train!`](@ref), [`evaluate!`](@ref), and [`predict!`](@ref), as well as in the following composite logging functions, which by default call the above primitives. Loggers may provide custom implementations of these.
 
 ```@docs
 Lighthouse.log_evaluation_row!
 log_values!
```

Comment:
```suggestion
log_value!
log_values!
```

Reply:
`log_value!` is earlier, in the "required" part of the interface